.github/workflows/deploy-published-releases.yaml

```diff
@@ -36,7 +36,7 @@ jobs:
           cp package.json LICENSE README.md build/
           cd build
           sed -i -e "s~\"version\": \"0.0.0-dev\"~\"version\": \"${GITHUB_REF##*/}\"~" package.json
-          sed -i -e "s~./build~.~" package.json
+          sed -i -e "s~./build/~./~" package.json
```

Comment:
I run the following command on my terminal:
```bash
sed -i -e "s~./build~.~" package.json
```

And the result was the following:
![image](https://user-images.githubusercontent.com/23644327/174610715-b7b2fe52-3b42-4b10-a11c-b6ef4126f37e.png)

That's why I ended up changing the regex. Do you have any suggestions on how the regex it can be improved?

Reply:
The `.` should be escaped: `\./build`. Just the dot matches any character on a regex